<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="horizontalWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>134</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <widget class="QCheckBox" name="checkBoxCreateSolid">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>121</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Solid</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>21</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>LX:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditLX">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>30 cm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>21</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>LY:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditLY">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>20 cm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>21</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>LZ:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditLZ">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>30</y>
        <width>71</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>40 cm</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Pipe name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Corner name:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSelectPipe">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>70</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Pipe</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonSelectCorner">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>111</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Corner</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPipeName">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>231</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCornerName">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>211</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelDraft">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../Resources/images/pipe-frame-box-dimensions.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSelectPipe</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>selectPipeClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSelectCorner</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>selectCornerClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>623</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectPipeClicked()</slot>
  <slot>selectCornerClicked()</slot>
 </slots>
</ui>
